cmd/glasskube/cmd/list.go

```diff
@@ -109,6 +146,36 @@ func printPackageTable(packages []*list.PackageWithStatus) {
 	}
 }
 
+func printPackageJSON(packages []*list.PackageWithStatus) {
+	enc := json.NewEncoder(os.Stdout)
+	enc.SetIndent("", "    ")
+	err := enc.Encode(packages)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error marshaling data to JSON: %v\n", err)
+		cliutils.ExitWithError()
+	}
+}
+
+func printPackageYAML(packages []*list.PackageWithStatus) {
+	var isFirstPackage = true
+
+	for _, pkg := range packages {
+		yamlData, err := yaml.Marshal(pkg)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "error marshaling data to YAML: %v\n", err)
+			cliutils.ExitWithError()
+		}
+
+		if isFirstPackage {
+			isFirstPackage = false
+		} else {
+			fmt.Println("---")
+		}
```

Comment:
I think this could be simplified by utilizing the loop counter instead, like:
```go
for i, pkg := range packages {
    /* ... */
    if i > 0 {
        fmt.Println("---")
    }
    /* ... */
}
```